src/components/Table/hoc/withTableSelection/withTableSelection.tsx

```diff
@@ -18,6 +18,7 @@ export interface WithTableSelectionProps<I> {
     onSelectionChange: (ids: string[]) => void;
     selectedIds: string[];
     isRowSelectionDisabled?: (item: I, index: number) => boolean;
+    disableRangeSelectionWithShift?: boolean;
```

Comment:
Not sure this prop really needed

Reply:
[b19c03c](https://github.com/gravity-ui/uikit/pull/374/commits/b19c03cfc800dabbb5eca539bed53422a16448fa)